modals/styles2-modals.css

```diff
@@ -0,0 +1,67 @@
+/* Modals Main */
+.modal-container {
+  margin: 3rem;
+}
+
+.modal-main-heading {
+  color: yellow;
+}
+.modal-snippet iframe {
+  margin: 1rem 0rem;
+  width: 564px;
```

Comment:
use rems instead of px, its better for accomplishing responsiveness faster

Reply:
ok